.circleci/config/commands/@yarn-cache.yml

```diff
@@ -0,0 +1,10 @@
+restore_yarn_cache:
```

Comment:
Note to reviewers: This file contains code migrated from `.circleci/config/@config.yml` See [this comment](https://github.com/hashicorp/vault/pull/9653/files#r465849114) (above)